test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_custom_target(ossfuzz)
+add_dependencies(ossfuzz solc_opt_target solc_noopt_target const_opt_target)
+
+add_executable(solc_opt_target solc_opt_target.cpp)
+target_link_libraries(solc_opt_target PRIVATE libsolc evmasm FuzzingEngine.a)
```

Comment:
Can you add a comment here that `FuzzingEngine.a` is "magic code" provided by google (at least I assume it is) when they compile it?

Reply:
Sure, will do.